addons/survey/views/survey_templates_print.xml

```diff
@@ -68,5 +108,18 @@
             </body>
         </html>
     </template>
+
+    <!-- simple scored fields -->
+    <template id="survey_scored_fields" name="Survey: scored fields">
+        <t t-if="question.question_type == 'numerical_box'"><t t-call="survey.question_numerical_box"/></t>
+        <t t-if="question.question_type == 'date'"><t t-call="survey.question_date"/></t>
+        <t t-if="question.question_type == 'datetime'"><t t-call="survey.question_datetime"/></t>
+    </template>
+
+    <!-- simple Char and multi text fields -->
+    <template id="survey_text_fields" name="Survey: Text and Char fields">
+        <t t-if="question.question_type == 'text_box'"><t t-call="survey.question_text_box"/></t>
+        <t t-if="question.question_type == 'char_box'"><t t-call="survey.question_char_box"/></t>
+    </template>
```

Comment:
I think I would still remove those.
They are a bit confusing since you are now calling "scored_fields" in a t-else when there is no scoring...

You can simply embed the calls and have a bit of copy/paste.
To shorten a bit you can do:
`<t t-if="question.question_type == 'numerical_box'" t-call="survey.question_numerical_box"/>`
instead of:
`<t t-if="question.question_type == 'numerical_box'"><t t-call="survey.question_numerical_box"/></t>`